system/HTTP/URI.php

```diff
@@ -593,8 +593,20 @@ public function getTotalSegments(): int
 	 */
 	public function __toString(): string
 	{
+		// If hosted in a sub-folder, we will have additional
+		// segments that show up prior to the URI path we just
+		// grabbed from the request, so add it on if necessary.
+		$baseUri  = new self(config(\Config\App::class)->baseURL);
+		$basePath = rtrim($baseUri->getPath(), '/') . '/';
+		$path     = $this->getPath();
+
+		if (! empty($basePath) && $basePath !== '/' && strpos($path, $basePath) !== 0)
```

Comment:
`$basePath` never be empty as always added with `/` so the empty check can be removed.

```suggestion
              if ($basePath !== '/' && strpos($path, $basePath) !== 0)
```